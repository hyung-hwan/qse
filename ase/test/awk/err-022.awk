global ARGV;

BEGIN {
	print ARGC;
}
