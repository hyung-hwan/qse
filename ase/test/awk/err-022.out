PARSE ERROR: CODE [62] LINE [1] duplicate global variable 'ARGV'
